config_test.go

```diff
@@ -0,0 +1,26 @@
+package wireless
+
+import (
+	"strconv"
+	"testing"
+)
+
+var configTests = []string{
+	"hello world",
+	`hello\world`,
+	"hello\tworld",
+	"Ë \" ' \\ 😅",
+}
+
+func TestConfigRoundTrip(t *testing.T) {
+	for _, tt := range configTests {
```

Comment:
Can you make these tests explicit? Automated tests can be hard to read and hard to figure out where the error is coming from

Reply:
Changed here: https://github.com/Plantiga/go-wireless/pull/2/commits/58650b5066c83c55d44752878cc996185923ffeb

I think I prefer the automated test, but I wouldn't insist on it.

Reply:
How about the named tests described here https://elliotchance.medium.com/go-data-driven-or-parameterized-tests-cd2b62b21c8a?

Also, I would like to see the tests test the intermediate value as well as the round trip, `\\t` => `\t` => `\\t` so that we don't miss anything.

Reply:
func TestConfigRoundTrip(t *testing.T) {
	for _, encoded, decoded := range configTests {
               t.Run("Test encode " + decoded, func(t *testing.T) { ...
               t.Run("Test decode " + encoded, func(t *testing.T) { ...